Check _LP64 instead of hardcoding a list of 64 bits archs, fixes build
on riscv64.

Index: extern/cuew/include/cuew.h
--- extern/cuew/include/cuew.h.orig
+++ extern/cuew/include/cuew.h
@@ -127,7 +127,7 @@ typedef uint32_t cuuint32_t;
 typedef uint64_t cuuint64_t;
 #endif
 
-#if defined(__x86_64) || defined(AMD64) || defined(_M_AMD64) || defined (__aarch64__) || defined(_M_ARM64) || defined(__ppc64__) || defined(__PPC64__)
+#if defined(_LP64)
 typedef unsigned long long CUdeviceptr;
 #else
 typedef unsigned int CUdeviceptr;
